packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -104,7 +106,7 @@ function InitialsAvatarFallback(props: AvatarProps) {
       <BoringAvatar
         size={Number(props.size?.(parsedInternalTheme))}
         name={`${props.firstName} ${props.lastName}`}
-        colors={avatarColors}
+        colors={avatarColors as string[]}
```

Comment:
In general, let's try to avoid explicit type assertions (`as string[]`) as much as possible. When you use a type assertion, you lose all benefits of the TS inference from that point on. For example, the above line will only fail during **runtime** if avatarColors takes an unexpected value since TS will not be able to throw an error in advance.

Reply:
If you implement the changes the other comments suggest, I think we can simply:
```suggestion
        colors={parsedInternalTheme.colors.$colorAvatarGradient}
```